### PR TITLE
Session view added

### PR DESCRIPTION
**Pull request by SaptakS**
Fixes https://github.com/fossasia/open-event-orga-server/issues/485

What is done in this PR:
- [x] View with frontend done to add new paper through form
- [x] Link sent to the invited speaker for adding paper details with proper hash
- [x] hash matched with user_id to confirm the link is correct.

### CODE REVIEW

**Comment by SaptakS** [comment]
@rafalkowalski @juslee @aditya1702 review.

File: open_event/views/admin/models_views/invite.py [hunk 1]
```diff
@@ -16,6 +16,8 @@ def create_view(self, event_id):
             event = DataGetter.get_event(event_id)
             if user:
                 DataManager.add_invite_to_event(user.id, event_id)
-                Helper.send_email_invitation(email, user.login, event.name)
+                hash = DataGetter.get_invite_by_user_id(user.id).hash
+                link = "http://127.0.0.1:5000/admin/events/" + str(event_id) + "/session/new/" + str(user.id) + "/" + str(hash) + "/"
```
Comment (rafalkowalski) [inline open_event/views/admin/models_views/invite.py]: please use format %

File: open_event/helpers/data.py [hunk 2]
```diff
@@ -126,6 +128,24 @@ def create_session(form, event_id, is_accepted=True):
...
+        :param event_id: Session belongs to Event by event id
+        """
+        new_session = Session(title=form["title"],
+                              subtitle="",
+                              description=form["description"],
+                              start_time=form["start_time"],
+                              end_time=form["end_time"],
+                              event_id=event_id,
+                              abstract="")
+
```
Comment (rafalkowalski) [inline open_event/helpers/data.py]: @SaptakS i think that we need to assign speaker to session, We need also state of session like, pending, accepted, rejected etc.

**Comment by rafalkowalski** [comment]
@SaptakS please add tests

File: open_event/helpers/data.py (hunk 2 above)
Comment (SaptakS) [inline open_event/helpers/data.py]: There is a field is_accepted which is false by default when an user submits a paper on invitation. We need to assign speaker. I will work on that.

**Comment by SaptakS** [comment]
Testing: https://github.com/fossasia/open-event-orga-server/issues/496
Displaying Session: https://github.com/fossasia/open-event-orga-server/issues/495

**Pull request merged**